dxlepoclient/client.py

```diff
@@ -250,9 +250,9 @@ def _sync_request(dxl_client, request, response_timeout, payload_dict):
             # Display the response
             logger.debug("Response:\n%s", ret_val)
             return ret_val
-        else:
-            raise Exception("Error: " + res.error_message + " (" + str(
-                res.error_code) + ")")
+
+        raise Exception("Error: " + res.error_message + " (" + str(
```

Comment:
Style comment - I think this might read a bit better if it were done like this instead:

```python
if res.message_type == Message.MESSAGE_TYPE_ERROR:
    raise Exception(...)

ret_val = MessageUtils.decode_payload(res)
...
return ret_val
```

It seems a little nicer to have the comparison be for equality instead of inequality. Also, with this approach, we wouldn't have to have an early return statement.